spec/std/openssl/ssl/server_spec.cr

```diff
@@ -135,9 +135,20 @@ describe OpenSSL::SSL::Server do
         end
       end
 
-      client = server.accept
-      client.hostname.should eq("example.com")
-      client.close
+      done = Channel(Nil).new
+      spawn do
+        client = server.accept
+        client.hostname.should eq("example.com")
```

Comment:
Does assertions work across fibers?

Reply:
I hope so? This by far isn't the first place to do  this, just look  20 lines up.